src/client/renderer/r_types.h

```diff
@@ -155,24 +157,26 @@ typedef enum {
  * up with the ones above to make things simple.
  */
 typedef enum {
-	R_ATTRIB_MASK_POSITION		= (1 << R_ATTRIB_POSITION),
-	R_ATTRIB_MASK_COLOR			= (1 << R_ATTRIB_COLOR),
-	R_ATTRIB_MASK_NORMAL		= (1 << R_ATTRIB_NORMAL),
-	R_ATTRIB_MASK_TANGENT		= (1 << R_ATTRIB_TANGENT),
-	R_ATTRIB_MASK_DIFFUSE		= (1 << R_ATTRIB_DIFFUSE),
-	R_ATTRIB_MASK_LIGHTMAP		= (1 << R_ATTRIB_LIGHTMAP),
-
-	R_ATTRIB_MASK_NEXT_POSITION	= (1 << R_ATTRIB_NEXT_POSITION),
-	R_ATTRIB_MASK_NEXT_NORMAL	= (1 << R_ATTRIB_NEXT_NORMAL),
-	R_ATTRIB_MASK_NEXT_TANGENT	= (1 << R_ATTRIB_NEXT_TANGENT),
+	R_ATTRIB_MASK_POSITION			= (1 << R_ATTRIB_POSITION),
+	R_ATTRIB_MASK_COLOR				= (1 << R_ATTRIB_COLOR),
+	R_ATTRIB_MASK_NORMAL			= (1 << R_ATTRIB_NORMAL),
+	R_ATTRIB_MASK_TANGENT			= (1 << R_ATTRIB_TANGENT),
+	R_ATTRIB_MASK_BITANGENT			= (1 << R_ATTRIB_BITANGENT),
+	R_ATTRIB_MASK_DIFFUSE			= (1 << R_ATTRIB_DIFFUSE),
+	R_ATTRIB_MASK_LIGHTMAP			= (1 << R_ATTRIB_LIGHTMAP),
+
+	R_ATTRIB_MASK_NEXT_POSITION		= (1 << R_ATTRIB_NEXT_POSITION),
+	R_ATTRIB_MASK_NEXT_NORMAL		= (1 << R_ATTRIB_NEXT_NORMAL),
+	R_ATTRIB_MASK_NEXT_TANGENT		= (1 << R_ATTRIB_NEXT_TANGENT),
+	R_ATTRIB_MASK_NEXT_BITANGENT	= (1 << R_ATTRIB_NEXT_BITANGENT),
 	
-	R_ATTRIB_MASK_SCALE			= (1 << R_ATTRIB_SCALE),
-	R_ATTRIB_MASK_ROLL			= (1 << R_ATTRIB_ROLL),
-	R_ATTRIB_MASK_END			= (1 << R_ATTRIB_END),
-	R_ATTRIB_MASK_TYPE			= (1 << R_ATTRIB_TYPE),
+	R_ATTRIB_MASK_SCALE				= (1 << R_ATTRIB_SCALE),
+	R_ATTRIB_MASK_ROLL				= (1 << R_ATTRIB_ROLL),
+	R_ATTRIB_MASK_END				= (1 << R_ATTRIB_END),
+	R_ATTRIB_MASK_TYPE				= (1 << R_ATTRIB_TYPE),
 
-	R_ATTRIB_MASK_ALL			= (1 << R_ATTRIB_ALL) - 1,
-	R_ATTRIB_GEOMETRY_MASK		= R_ATTRIB_MASK_SCALE | R_ATTRIB_MASK_ROLL | R_ATTRIB_MASK_END | R_ATTRIB_MASK_TYPE
+	R_ATTRIB_MASK_ALL				= (1 << R_ATTRIB_ALL) - 1,
+	R_ATTRIB_GEOMETRY_MASK			= R_ATTRIB_MASK_SCALE | R_ATTRIB_MASK_ROLL | R_ATTRIB_MASK_END | R_ATTRIB_MASK_TYPE
```

Comment:
Watch the whitespace diffs here.

Reply:
For a reasons it looked fine in Visual Studio. Mine tab size is 4. Will check it in another editor as well.

Reply:
We use hard tabs, maybe that's the issue.